docs/stdlib/json.rst

```diff
@@ -289,20 +285,20 @@ possible to cast a JSON value directly into a :eql:type:`tuple`.
 
     :index: array unpack
 
-    Return elements of JSON array as a set of :eql:type:`json`.
+    Returns the elements of a JSON array as a set of :eql:type:`json`.
 
-    Calling this function on anything other than a JSON array will
-    cause a runtime error.
-
-    This function should be used if the ordering of elements is not
-    important or when set ordering is preserved (such as an immediate
-    input to an aggregate function).
+    Calling this function on anything other than an array of :eql:type:`json`
+    will result in a runtime error:
```

Comment:
```suggestion
    Calling this function on anything other than an :eql:type:`json` array will
    result in a runtime error.
```

Revert this for two reasons:

- This operates on a `json` array, not an array of `json`
- The code example demonstrates that ordering is not guaranteed, which is not the subject of this sentence.

As a result, we also need to move the paragraph that begins "This function should be..." back above the code block. Your rewrite of the paragraph is good with some small tweaks: 

```rest
    This function should be used only if the ordering of elements is not
    deemed important or when the ordering of the set is preserved (such as an
    immediate input to an aggregate function).
```
- Pair "only" with the limiting factor (i.e., "if the ordering of elements is not deemed important")
- "ordering is" instead of "ordering are". Since "ordering" is singular, the verb should be too.
- End with a period